docs/docs/bootstrapping.md

```diff
@@ -39,9 +39,9 @@ In this async mode, non-bootstrapped tables are replicated as normal by the main
 ***
 
 * a bootstrap starts with a document with `type = "bootstrap-start"`
```

Comment:
i would change all of these to read "starts with an event of type 'bootstrap-...'"